experimental/go-dep/image/Dockerfile-stack

```diff
@@ -0,0 +1,40 @@
+FROM golang:1
+
+ENV APPSODY_MOUNTS=/:/project/user-app
+ENV APPSODY_DEPS=
+
+ENV APPSODY_WATCH_DIR=/project/user-app
+ENV APPSODY_WATCH_IGNORE_DIR=/project/user-app/vendor
+ENV APPSODY_WATCH_REGEX="^.*.(go|toml)$"
+
+ENV APPSODY_INSTALL="bash /vendor.sh"
+
+ENV APPSODY_RUN="go run ."
+ENV APPSODY_RUN_ON_CHANGE="go run ."
+ENV APPSODY_RUN_KILL=false
+
+ENV APPSODY_DEBUG="dlv debug --listen 0.0.0.0:9229 --headless ."
+ENV APPSODY_DEBUG_ON_CHANGE="dlv debug --listen 0.0.0.0:9229 --headless ."
+ENV APPSODY_DEBUG_KILL=true
+
+ENV APPSODY_TEST="go test ."
+ENV APPSODY_TEST_ON_CHANGE=""
```

Comment:
nit: If `APPSODY_TEST_ON_CHANGE` isn't being used here can this be removed?

Reply:
Resolved with https://github.com/appsody/stacks/pull/425/commits/0d3b961aff58799f6aefb41d338d50b75769ee7c